fastparquet/test/test_util.py

```diff
@@ -5,25 +5,21 @@
 
 def test_analyse_paths():
     file_list = ['a', 'b']
-    base, out = analyse_paths(file_list, '/')
-    assert (base, out) == ('', ['a', 'b'])
+    base, out = analyse_paths(file_list)
+    assert (base, out) == ('.', ['a', 'b'])
 
     file_list = ['c/a', 'c/b']
-    base, out = analyse_paths(file_list, '/')
+    base, out = analyse_paths(file_list)
     assert (base, out) == ('c', ['a', 'b'])
 
     file_list = ['c/d/a', 'c/d/b']
-    base, out = analyse_paths(file_list, '/')
+    base, out = analyse_paths(file_list)
     assert (base, out) == ('c/d', ['a', 'b'])
 
     file_list = ['c/cat=1/a', 'c/cat=2/b', 'c/cat=1/c']
-    base, out = analyse_paths(file_list, '/')
+    base, out = analyse_paths(file_list)
     assert (base, out) == ('c', ['cat=1/a', 'cat=2/b', 'cat=1/c'])
 
-    file_list = ['c\\cat=1\\a', 'c\\cat=2\\b', 'c\\cat=1\\c']
-    base, out = analyse_paths(file_list, '\\')
-    assert (base, out) == ('c', ['cat=1\\a', 'cat=2\\b', 'cat=1\\c'])
```

Comment:
This test was removed because program is standardized on `/` as a separator.

Reply:
To be sure: if the user provides windows-like paths to `ParquetFile`, do they get converted before being passed down to explicitly '/'-based path functions?

Reply:
Yes. I added a specific test that will send os-specific file name.  I also reverted the testfile() fixture so all files in testing are os-specific.  All tests pass on Windows.